lib/bibliothecary/related_files_info.rb

```diff
@@ -1,7 +1,6 @@
 module Bibliothecary
   class RelatedFilesInfo
-    attr_reader :folder_path
-    attr_reader :relative_folder_path
+    attr_reader :path
```

Comment:
so which path is this? the relative subfolder path that the related files are inside?

Reply:
if so maybe leave it named folder_path but make it always relative (don't need the absolute one at all?)

Reply:
Yep. I figure you called the thing with the base path of what you're looking at, so you would expected everything to be relative to that. (I would expect that)

Reply:
I'd agree if we were looking at a *file* but we're basically looking at a folder, so it feels sort of redundant. But I'm not super committed.

Reply:
"The *path* of the files"

Reply:
wfm :-) let's not block on over-perfecting this